docs/make_docs_without_docker.sh

```diff
@@ -21,12 +21,12 @@ rm -rf autogenerated_source
 rm -rf build
 
 # generate content
-python generate_src_rst_files.py  --root_dir ../ivy || exit 1
+python3 generate_src_rst_files.py  --root_dir ../ivy || exit 1
```

Comment:
This shouldn't be `python3` but `python` instead as the dockerless pipeline is aimed at working with a virtual environment where `python` is used. Same applies to the other updates from `python` to `python3` in this file